### PR TITLE
Added basic Node script that outputs json file.

### PR DESCRIPTION
**Pull request by jakesmithdeveloper**
Uses command line arguments example below
node txt_to_json | PodcastName | Episode Number | input file.txt | output file.json

### CODE REVIEW

File: txt_to_json.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+				transcript.push({text:line});
+			} else if(!isNaN(parseInt(line[0], 10))) {
+				// Timestamp and text
+				transcript.push({timestamp:line.slice(0,9), text:line.slice(10,line.length)});
+			} else {
+				console.log('Error: Unknown format on line '+count+': '+"\""+line+"\"");
+			}
+		}
+		lr.resume();
+	}, 100);
```
Comment (statianzo) [inline txt_to_json.js]: Why is a 100ms timeout necessary here?

File: txt_to_json.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+});
+
+lr.on('end', function () {
+	// All lines are read, file is closed now.
+	let json_object = {podcast:process.argv[2],episode:process.argv[3],"transcript":transcript}
+	var fs = require('fs');
+	fs.writeFile(process.argv[5], JSON.stringify(json_object), 'utf8');
```
Comment (statianzo) [inline txt_to_json.js]: `JSON.stringify(json_object, null, 2)` will write formatted JSON as output. The `2` represents using 2 spaces for indentation.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Syntax

**Comment by agarrharr** [comment]
How do you run it?

**Comment by agarrharr** [comment]
I get the error: `Error: Cannot find module 'line-by-line'`

**Comment by agarrharr** [comment]
Thanks, I made a few changes and merged our PR! Nice work! 🎉 This will definitely save me some time!

**Comment by agarrharr** [comment]
Closes #106

**Pull request merged**